plugins/forc-index/src/ops/forc_index_auth.rs

```diff
@@ -95,7 +95,7 @@ pub fn init(command: AuthCommand) -> anyhow::Result<()> {
     let response: SignatureResponse = res.json().unwrap();
 
     if let Some(token) = response.token {
-        info!(
+        println!(
```

Comment:
So this issue is 2 parts:
1. We want to hide (most) output behind a `-v --verbose` flag
2. The existing info that we're logging (e.g., this `print`) we want to make less verbose
  - This `print` is a _really_ long informative string. That's ok if `-v --verbose` is passed, but if not, it should just say something like `"✅ Authenticated successfully"`
  - Also this should remain `info` and not `println`